index.html

```diff
@@ -1088,6 +1089,257 @@ <h4>Nonce</h4>
+<a>canonicalization algorithm</a>.
+                </li>
+              </ol>
+            </li>
+            <li>
+Return the <a>statement digests</a> array and the recipient binding commitment.
+            </li>
+          </ol>
+
+        </section>
+
+      </section>
+
+      <section>
+        <h3>Terms</h3>
+
+        <p>
+The following section outlines the terms used by the BBS+ Bound Signature Suite
+which differ from those terms used by the BBS+ Signature Suite. All other terms
+are the same in both suites.
+        </p>
+
+        <section>
+          <h4>Proof Type</h4>
+
+          <p>
+To identify the type of <a>linked data proof</a> that is attached to a
+<a>linked data document</a>, use the <code>type</code> attribute
+<a href="https://w3c-ccg.github.io/ld-proofs/#linked-data-signatures">defined</a>
+in [[LD-PROOFS]].
+          </p>
+
+          <p>
+The term of <code><dfn>BbsBlsBoundSignature2020</dfn></code> is used to indicate
+when a <a>linked data proof</a> is of type BBS+ Bound Signature.
+          </p>
+
+          <p>
+A <a>linked data document</a> featuring a BBS+ Bound Signature
+<a>linked data proof</a> MUST contain a proof element that has a type equal to
+<code>BbsBlsBoundSignature2020</code>.
+          </p>
+
+          <pre class="example nohighlight">
+            {
+              "@context": [
+                "http://schema.org/",
+                "https://w3id.org/security/v2",
+                "https://w3id.org/security/bbs/v1"
+              ],
+              "@type": "Person",
+              "firstName": "Jane",
+              "lastName": "Does",
+              "jobTitle": "Professor",
+              "telephone": "(425) 123-4567",
+              "email": "jane.doe@example.com",
+              "proof": {
+                <span class="highlight">"type": "BbsBlsBoundSignature2020"</span>,
+                "created": "2020-04-25",
+                "verificationMethod": "did:example:489398593#test",
+                "proofPurpose": "assertionMethod",
+                "proofValue": "F9uMuJzNBqj4j+HPTvWjUN/MNoe6KRH0818WkvDn2Sf7kg1P17YpNyzSB+CH57AWDFunU13tL8oTBDpBhODckelTxHIaEfG0rNmqmjK6DOs0/ObksTZh7W3OTbqfD2h4C/wqqMQHSWdXXnojwyFDEg=="
+              }
+            }
+          </pre>
+        </section>
+      </section>
+    </section>
+
+    <section>
+      <h2>The BBS+ Bound Signature Proof Suite 2020</h2>
+      <p>
+A BBS bound proof of knowledge <a>linked data proof</a> is a proof that is
+derived from a <a>BbsBlsBoundSignature2020</a> <a>linked data proof</a> where
+a sub-set of the original <a>statements</a> are revealed.
+      </p>
+      <p>
+The BBS+ bound proof of knowledge <a>signature suite</a> MUST be used in
+conjunction with the signing and verification algorithms in the Linked Data
+Proofs [[LD-PROOFS]] specification. It is identical to the BBS+ proof of
+knowledge <a>signature suite</a> with the addition of a proof of recipient
+binding. The suite consists of the following algorithms:
```

Comment:
```suggestion
Proofs [[LD-PROOFS]] specification. It adds a proof of recipient binding,
and is otherwise identical, to the BBS+ proof of knowledge <a>signature
suite</a>. The suite consists of the following algorithms:
```